src/symmeplot/matplotlib/scene.py

```diff
@@ -144,6 +145,51 @@ def plot(self, prettify=True, ax_scale=1.5):
             self.auto_zoom()
             self.axes.set_aspect("equal", adjustable="box")
 
+    def set_plot_as_2d(self, frame: ReferenceFrame | None = None) -> None:
+        """Change the axis to an orhogonal projection making the view seemingly 2D.
+
+        Parameters
+        ----------
+        frame : ReferenceFrame, optional
+            Reference frame w.r.t. which the axis view is oriented aligning the users
+            view with the XY plane. The default is the inertial frame of the scene.
+
+        """
+        projection_frame = ReferenceFrame("A")
+        projection_frame.orient_axis(self.inertial_frame, self.inertial_frame.z, 0)
+        frame = frame or projection_frame
```

Comment:
Why do you create a `projection_frame` here in the same orientation as the `intertial_frame`? I would expect that you can directly use the inertial frame, as you also do in one of the tests.

Reply:
I think this was a left over of an implementation I had where `frame` could not be the `self.interial_frame`. It can indeed be removed.